.cirrus.yml

```diff
@@ -11,4 +11,4 @@ task:
     - cargo build
   test_script:
     - . $HOME/.cargo/env
-    - cargo test
+    - cargo test -p uucore -p coreutils
```

Comment:
Basically implicitly we always run `cargo test -p coreutils` and by specifying to also include the project `-p uucore` we test more things in the same run

It is important to run both at the same time, because otherwise the feature flags enabled by compiling `coreutils` are not passed to `uucore`. 

As an example, the `uucore/fs` tests are only ran if anything in `coreutils` enables the fs feature. Which this we piggyback upon the wide array of tests and conditional features being tested for all the platforms they are valid on and don't have to develop a second set of feature flags for `uucore` itself